WordPress/Sniffs/VIP/SlowDBQuerySniff.php

```diff
@@ -42,6 +42,7 @@ public function getGroups() {
 	 * Processes this test, when one of its tokens is encountered.
 	 *
 	 * @since 0.10.0
+	 * @since 0.12.0 Added the 'slow query' comment.
```

Comment:
This sounds a little non-descript. What about something like _Introduced new `slow_query` whitelist comment_ ?